pkgs/tools/networking/maubot/default.nix

```
@@ -0,0 +1,100 @@
+{ lib
+, fetchpatch
+, python3
+, runCommand
+, encryptionSupport ? true
+}:
+
+let
+  python = python3.override {
+    packageOverrides = self: super: {
+      sqlalchemy = super.buildPythonPackage rec {
+        pname = "SQLAlchemy";
+        version = "1.3.24";
+
+        src = super.fetchPypi {
+          inherit pname version;
+          sha256 = "sha256-67t3fL+TEjWbiXv4G6ANrg9ctp+6KhgmXcwYpvXvdRk=";
+        };
+
+        postInstall = ''
+          sed -e 's:--max-worker-restart=5::g' -i setup.cfg
+        '';
+
+        doCheck = false;
+      };
+    };
+  };
+
+  self = with python.pkgs; buildPythonApplication rec {
+    pname = "maubot";
+    version = "0.4.1";
+    disabled = pythonOlder "3.8";
+
+    src = fetchPypi {
+      inherit pname version;
+      sha256 = "sha256-Ro2PPgF8818F8JewPZ3AlbfWFNNHKTZkQq+1zpm3kk4=";
+    };
+
+    patches = [
+      # add entry point
+      (fetchpatch {
+        url = "https://patch-diff.githubusercontent.com/raw/maubot/maubot/pull/146.patch";
+        sha256 = "0yn5357z346qzy5v5g124mgiah1xsi9yyfq42zg028c8paiw8s8x";
+      })
+      ./allow-building-plugins-from-nix-store.patch
```

Usually, these are in a separate dir.

```suggestion
      ./patches/allow-building-plugins-from-nix-store.patch
```

---

```
$ find | grep \\.patch | wc -l
3227
$ find | grep \\.patch | grep /patches | wc -l
155
```

---

I stand corrected!